iFixit/Bootstrap-Form-Builder
# Form Builder: Make all labels textareas

We already made this change for radio buttons [here](https://github.com/iFixit/Bootstrap-Form-Builder/pull/14) since we wanted them to have a max character limit of 255. Now we want this change for all the inputs.

QA
===
- Setup in 3P/Bootstrap-Form-Builder:
   - `git pull` and `checkout` this branch
   - Run `make`
- Add one of every input type to a datacapture form.
- Verify they all have textareas for the "Label" field.

Part of iFixit/dozuki#2010.
